kong/globalpatches.lua

```diff
@@ -589,6 +591,20 @@ return function(options)
       return sock
     end
 
+    -- OTel-formatted logs feature
+    local dynamic_hook = require "kong.dynamic_hook"
+    _G.ngx.log = function(...)
```

Comment:
We still need logs something inside the o11y framework, maybe we can export `_G.native_ngx_log` for this usage.

And we can discuss if we need to ship the log inside the framework by OTLP.

Reply:
done.